### PR TITLE
Accept Backtab and Shift-Tab for hovering selection backwards

### LINKED ISSUE #690: Allow hovering list items backwards with Shift-Tab
### The Problem

[It's possible to navigate list items forward with `Tab`](https://github.com/Nheko-Reborn/nheko/search?q=Qt.Key_Tab), but not backwards with `Shift-Tab`.

### The Solution

`Shift-Tab` would be the counterpart of `Tab` for hovering the previous item. This is how it works in e.g. web browsers, as well as many other apps in the wild.

### Alternatives

_No response_

### Additional context

_No response_

### Happens in the latest version

- [X] Yes, this feature is still missing.

### PR DESCRIPTION
**Pull request by resolritter**
closes #690

### CODE REVIEW

**Comment by deepbluev7** [comment]
Awesome! Would you mind also adding that to the MessageInput, or would you rather just add those 2? You probably just need to add the shift check here: https://github.com/Nheko-Reborn/nheko/blob/master/resources/qml/MessageInput.qml#L198

**Comment by resolritter** [comment]
> Awesome! Would you mind also adding that to the MessageInput

Done. Sorry I missed it the first time.

**Comment by deepbluev7** [comment]
No worries, thanks for implementing this! I'll test it locally and then this should be good to merge. Thanks a lot!

File: resources/qml/MessageInput.qml [hunk 1]
```diff
@@ -195,7 +195,11 @@ Rectangle {
                     } else if (event.key == Qt.Key_Tab) {
                         event.accepted = true;
                         if (popup.opened) {
-                            popup.up();
+                            if (event.key == Qt.Key_Tab && (event.modifiers & Qt.ShiftModifier)) {
```
Comment (resolritter) [inline resources/qml/MessageInput.qml:198]: oops I just noticed this `event.key == Qt.Key_Tab` part is not needed (already checked above), I'll rectify this

**Comment by deepbluev7** [comment]
Seems to work great, thanks a lot!

**Pull request merged**